documentation/adrs/static-analysis-baseline.md

```diff
@@ -0,0 +1,50 @@
+# Static Analysis Baseline
+
+Proposed by: @norberttech
+Date: 2025-01-07
```

Comment:
What I miss is: `Status: proposal/accepted/denied` etc.

Reply:
There is an explanation already in the ADR's readme that any ADR that is merge is considered accepted, propals are open PR's, denied are closed PR - adding status here IMO is redundant. 

My goal with this template is to make it as minimalistic as possible and as descriptive as possible, that's why it does not come with all the sections that are mentioned by @Bellangelo

Reply:
Alternatively for denied there could be list of PRs with specific tag and link to that list in ADR list, so it would be easily to find if someone already proposed something that got denied, wdyt?

Reply:
hmm we could put a label "ADR" and a link to all closed PRs with label ADR under index od merged ADRs as "Rejected ADRs"

Reply:
That's what I had in mind 👍

Reply:
added 🤝